src/process/dms/CCPr_DMS_Mod.F90

```diff
@@ -0,0 +1,299 @@
+!> \brief Driver for CATChem DMS process
+!!
+!!\defgroup catchem_dms_process
+!! The CATChem DMS Process group holds all the CATCHem DMS processes.
+!!
+!! \author Lacey Holland and Wei Li
+!! \date 01/2025
+!!!>
+MODULE CCPR_DMS_mod
+   USE Precision_mod
+   USE Error_Mod
+   USE DiagState_Mod, Only : DiagStateType
+   USE MetState_Mod,  Only : MetStateType
+   USE ChemState_Mod, Only : ChemStateType
+   USE EmisState_Mod,  Only : EmisStateType
+   USE GridState_Mod,  Only : GridStateType
+   USE Config_Opt_Mod, Only : ConfigType
+
+   IMPLICIT NONE
+
+   PRIVATE
+
+   PUBLIC :: CCPR_DMS_Init
+   PUBLIC :: CCPR_DMS_Run
+   PUBLIC :: CCPR_DMS_Finalize
+   PUBLIC :: DMSStateType
+
+   !> \brief DMSStateType
+   !!
+   !! \details Contains all the information needed to run DMS Process
+   !!
+   !! This type contains the following variables:
+   !! \param Activate Activate Process (True/False)
+   !! \param SchemeOpt Scheme Option
+   !! \param nDMSSpecies Number of DMS species
+   !! \param DMSSpeciesIndex Index of DMS species
+   !! \param DMSSpeciesName Name of DMS species
+   !! \param SpcIDs CATChem species IDs
```

Comment:
What is this SpcIDs term? I see why you need the DMS number of species, index of DMS species, and name of DMS species. But what exactly are we storing as SpcIDs?